parse.go

```diff
@@ -131,16 +131,20 @@ type Context struct {
 type Option func(*Context)
 
 // WithConfigFile tells Parse to read the provided filename as a config file.
-// Requires WithConfigFileParser, and overrides WithConfigFileFlag.
+// Requires WithConfigFileParser, and overrides WithConfigFileFlag. If you
+// want both a config file flag and a default value when one is unprovided,
+// use WithConfigFileFlag and set your chosen default when defining the
+// flag.
```

Comment:
👍

Reply:
I would even maybe add another sentence somewhere, like

> Because config files should generally be user-specifiable, this option should be rarely used in practice. Prefer WithConfigFileFlag.

Reply:
A+